packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -35,6 +32,33 @@
 	}
 }
 
+.block-editor-global-styles__shadow-dropdown {
+	> .components-button.has-icon.has-text:first-child {
+		gap: $grid-unit-10;
+	}
```

Comment:
CSS selectors with the `.components-` prefix should be avoided as much as possible. Can we use the `HStack` component inside the Button component as in the previous implementation?